test/onnx/test_models.py

```diff
@@ -189,5 +189,25 @@ def test_qat_resnet(self):
 
         self.exportTest(toC(qat_resnet50), toC(x))
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    def test_qat_resnet_per_channel(self):
+        # Quantize ResNet50 model
+        x = Variable(torch.randn(BATCH_SIZE, 3, 224, 224).fill_(1.0))
```

Comment:
Is explicit `Variable` still useful?

Reply:
I guess no. I copied from other tests. Will remove it.

Reply:
I would suggest to remove the deprecated usage of `Variable`. The other tests could be updated in a separate PR.

Reply:
Removed `Variable` in all fake quantization ONNX export tests.